src/frontend/src/lib/utils/format.utils.ts

```diff
@@ -29,8 +31,7 @@ export const formatToken = ({
 	if (trailingZeros) {
 		return formatted;
 	}
-
-	return formatted.replace(/\.0+$/, '');
+	return (showPlusSign && +res > 0 ? '+' : '') + formatted;
```

Comment:
I don't think spontanously we should modify `formatToken` with this. I'm a bit afraid of the side effect. Also not fully convinced that it should find place here.

Are you sure we won't break anything?

Reply:
I included some tests and the existing ones are quite extensive (I re-checked them). So, I think we are covered on this (`showPlusSign` defaults to `false` too)

However I understand your point: i was wondering indeed if it should stay all in the component or not; but I thought this could be a cool feature to have in the util, boh

Undecided eheh! Any strong opinion?

Reply:
I don't like it but, can live with it (particurlaly as I don't have to live too much with it 😉).